app/src/main/java/com/eveningoutpost/dexdrip/UtilityModels/Notifications.java

```diff
@@ -232,6 +233,15 @@ public void notificationSetter(Context context) {
         if (bg_ongoing && (Build.VERSION.SDK_INT >= Build.VERSION_CODES.JELLY_BEAN)) {
             bgOngoingNotification(bgGraphBuilder);
         }
+
+        List<BgReading> bgReadings = BgReading.latest(3);
+        if(bgReadings == null || bgReadings.size() < 1) { return; }
+        BgReading bgReading = bgReadings.get(0);
+
+        //Text to speech
+        Log.d("BgToSpeech", "gonna call speak");
+        BgToSpeech.getSingleton(context).speak(bgReading.calculated_value, bgReading.timestamp);
```

Comment:
Any reason this is here in the notifications and not in the BgSendQueue (where I figure all new values are going to be pushed through)
This however may get called at unexpected times, no?